UnitTests/BraintreePayPalTests/BTPayPalAccountNonce_Tests.swift

```diff
@@ -1,33 +1,9 @@
 import XCTest
 @testable import BraintreePayPal
 
-class BTPaymentMethodNonceParser_PayPal_Tests: XCTestCase {
-    func testSharedParser_whenTypeIsPayPal_returnsPayPalAccountNonce() {
-        let payPalAccountNonce = BTPayPalAccountNonce(
-            json: BTJSON(
-                value: [
-                    "consumed": false,
-                    "description": "jane.doe@example.com",
-                    "details": [
-                        "email": "jane.doe@example.com",
-                    ],
-                    "isLocked": false,
-                    "nonce": "a-nonce",
-                    "securityQuestions": [] as [Any?],
-                    "type": "PayPalAccount",
-                    "default": true
-                ] as [String: Any]
-            )
-        )
-
-        XCTAssertEqual(payPalAccountNonce?.nonce, "a-nonce")
-        XCTAssertEqual(payPalAccountNonce?.type, "PayPal")
-        XCTAssertEqual(payPalAccountNonce?.email, "jane.doe@example.com")
-        XCTAssertTrue(payPalAccountNonce!.isDefault)
-        XCTAssertNil(payPalAccountNonce?.creditFinancing)
-    }
+final class BTPayPalAccountNonce_Tests: XCTestCase {
```

Comment:
Renamed this class and the tests since they were not actually testing the shared parser and the test names were not accurate.